src/eval.h

```diff
@@ -3,15 +3,17 @@
 
 #define EVAL_MAX_VARIABLE_NAME_LENGTH 300
 
+/** List of `struct rrdvar`*/
 typedef struct eval_variable {
-    char *name;
-    uint32_t hash;
-    struct rrdvar *rrdvar;
-    struct eval_variable *next;
+    char *name;                 ///< Uniqe name
+    uint32_t hash;              ///< Hash of `name`
+    struct rrdvar *rrdvar;      ///< The variable
+    struct eval_variable *next; ///< Next item in the list
 } EVAL_VARIABLE;
 
+/// ktsaou: Your help needed
 typedef struct eval_expression {
-    const char *source;
+    const char *source;q
     const char *parsed_as;
```

Comment:
the expression, reformatted after parsing it (and generating the expression tree) and re-producing from the tree.